framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2104,7 +2171,7 @@ public boolean shouldSuppressWarnings(Tree tree, String errKey) {
         }
 
         // trees.getPath might be slow, but this is only used in error reporting
```

Comment:
This comment can be removed or adapted now?

Reply:
Yes, I have removed the comment since it seems having a treepath is necessary for the logic.